src/pages/SignIn.js

```diff
@@ -94,9 +95,11 @@ function SignIn() {
               setSex(event.target.value);
             }}
           />
-
-          <Input
-            placeholder="Wie lang ist deine Angel?"
+          <DistanceSlider
+            type="range"
```

Comment:
You already set `type`, `min` and `max` in `DistanceSlider`. If you like to override these properties, it's fine to give them a different value. 
But if you would change `type="range"`, it's not working like expected. Thus, you shouldn't set it here again.